design/src/rfcs/rfc0027_new_home_for_cred_cache_and_builder_api_consistency.md

```diff
@@ -0,0 +1,146 @@
+RFC: Finding New Home for Credential Types
+===================================================
+
+> Status: Implemented
+>
+> Applies to: clients
+
+This RFC supplements [smithy-rs#1842](https://github.com/awslabs/smithy-rs/pull/1842) and discusses for the selected design where to place the types for credentials providers, credentials caching, and everything else that comes with them.
```

Comment:
This one is merged now, so you can link to it directly.

Reply:
Thanks for pointing it out. Addressed in a298edb.